docs/using/backup_mongo.md

```diff
@@ -0,0 +1,170 @@
+8. If you're creating a schedule backups, also specify the backup type, the schedule, and a retention policy for your backup:
+    - **Backup Type**: currently, PMM supports both **Full** and Point-in-type recovery **(PITR)** backup types for MongoDB. However, the PITR option is only available for the **Logical** data model. For MySQL, only the **Full** type is supported.
+    - **Shedule**: configure the frequency and the start time for this backup. Make sure that the the schedule you specify here does not create overlapping jobs or overhead on the production environment. Also check that your specified shedule does not overlap with production hours.
+    - **Retention**: this option is only available for Snapshot backups stored on Amazon S3. If you wand to keep an unlimited number of backup artifacts, type `0`.
+9. Expand **Advanced Settings** to specify the settings for retrying the backup in case of any issues. You can either let PMM retry the backup again (**Auto**), or do it again yourself **Manual**. Auto retry mode enables you to select up to ten retries and an interval of up to eight hours between retries.
+10. Click **Backup** to start creating the backup artifact or schedule a job.
+11. Go to the **All Backups** tab, and check the **Status** column. An animated ellipsis indicator {{icon.bouncingellipsis}} shows that a backup is currently being created.
+
+## Edit a scheduled backup
+
+1. Go to **Backup > Scheduled Backup Jobs**.
+2. In the **Actions** column:
+    - Click the switch <i class="uil uil-toggle-on"></i> to enable or disable the backup.
+    - Click ![](../_images/dots-three-vertical.png) to edit, delete or create a (by default, disabled) copy of the backup schedule.
+
+## Restore a backup
+
+### Restore compatibility
+
+MongoDB backups can only be restored to the same service they were created from.
+
+To restore a backup:
+
+1. Go to <i class="uil uil-history"></i> **Backup > All backups** and find the backup that you want to restore.
+2. Click the arrow in the **Actions** column to check all the information for the backup, then click ![](../_images/dots-three-vertical.png) **> Restore from backup**.
+3. In the **Restore from backup** dialog, select **Same service** to restore to a service with identical properties or **Compatible services** to restore to a compatible service.
+4. Select one of the available service names from the drop-down menu.
+5. If you are restoring a PITR backup, also  select the point for the date and time that you want to restore the database to.
+6. Check the values, then click **Restore**.
+7. Go to the **Restores** tab to check the status of the restored backup.
+During restoring, PMM disables all the scheduled backup tasks for the current service. Remember to re-enable them manually after the restore.
+
+### Restore to a new cluster manually
```

Comment:
Not sure about word **cluster**. We're supporting only replica sets. Cluster is more common term that includes also sharded setups.

Reply:
per https://www.mongodb.com/basics/clusters
The Cluster's both ReplicaSet or Sharded.  
Since we have plans to add  Sharded, too - it's better to keep the generic name here